test/typescript.ts

```diff
@@ -1,87 +1,84 @@
-/// <reference types="mocha" />
```

Comment:
AFAIK triple-slash-directives should only be included in declaration files, not in actual code. The [TypeScript Handbook](http://www.typescriptlang.org/docs/handbook/triple-slash-directives.html) has this to say of the matter:

```
Use these directives only when you’re authoring a d.ts file by hand.
```

Reply:
I mean that's fine to say, but it used to be that removing them would break things and an import wasn't an option.  From my local copy that still has the deps from when this was written I get the following after removing them:
```
          throw new TSError(formatDiagnostics(diagnosticList, cwd, ts, lineOffset))
                ^
TSError: ⨯ Unable to compile TypeScript
test\typescript.ts (2,14): Cannot find name 'require'. (2304)
test\typescript.ts (3,24): Cannot find name 'require'. (2304)
test\typescript.ts (13,1): Cannot find name 'describe'. (2304)
test\typescript.ts (14,2): Cannot find name 'describe'. (2304)
test\typescript.ts (15,3): Cannot find name 'it'. (2304)
test\typescript.ts (26,3): Cannot find name 'it'. (2304)
test\typescript.ts (37,3): Cannot find name 'it'. (2304)
test\typescript.ts (49,2): Cannot find name 'describe'. (2304)
test\typescript.ts (50,3): Cannot find name 'it'. (2304)
test\typescript.ts (61,3): Cannot find name 'it'. (2304)
test\typescript.ts (73,3): Cannot find name 'it'. (2304)
```

It does look like it's fine with the latest versions though so I'm happy to remove them